system_setup/client/controllers/wslconfadvanced.py

```diff
@@ -33,21 +33,24 @@ async def make_ui(self):
     def run_answers(self):
         if all(elem in self.answers for elem in
                ['interop_enabled', 'interop_appendwindowspath',
-                'gui_theme', 'gui_followwintheme', 'legacy_gui',
-                'legacy_audio', 'adv_ip_detect',
-                'wsl_motd_news', 'automount', 'mountfstab']):
+                'gui_theme', 'gui_followwintheme', 'interop_guiintegration',
+                'interop_audiointegration', 'interop_advancedipdetection',
+                'motd_wslnewsenabled', 'automount_enabled',
+                'automount_mountfstab']):
             reconfiguration = WSLConfigurationAdvanced(
                 interop_enabled=self.answers['interop_enabled'],
```

Comment:
Is there a way to make all those assignements, now that there is a one to one mapping easier?
using a **kwargs and passing the dict or something along that idea?
(basically all those lines would become: `reconfiguration = WSLConfigurationAdvanced(self.asnwers)` on the caller side

Reply:
Just tested and you can just do:
```
reconfiguration = WSLConfigurationAdvanced(**self.answers)
```

Reply:
done